src/features/chart/components/Chart.tsx

```diff
@@ -25,9 +28,9 @@ const tenors = [
   { value: "max", label: "MAX" }
 ];
 
-const yaxisFormat = item => item.toFixed(2);
+const yaxisFormat = (item: number) => item.toFixed(2);
 
-const formatDate = (isoDate, ChartRange) => {
+const formatDate = (isoDate: Date, ChartRange: string) => {
```

Comment:
q: Isn't `isoDate` a string that's sent from the server, which is then converted into a `Date`?

Reply:
Very good spot!